src/GeneratedInjectorDelegator.php

```diff
@@ -22,7 +22,9 @@ public function __invoke(ContainerInterface $container, $name, callable $callbac
     {
         $config = $container->has('config') ? $container->get('config') : [];
         $aotConfig = $config['dependencies']['auto']['aot'] ?? [];
-        $namespace = empty($aotConfig['namespace']) ? 'Laminas\Di\Generated' : $aotConfig['namespace'];
+        $namespace = ! isset($aotConfig['namespace']) || $aotConfig['namespace'] === ''
+            ? 'Laminas\Di\Generated'
+            : $aotConfig['namespace'];
```

Comment:
Is there an effective way to test this behavior? 

Also, code above can probably be rewritten as `$namespace = ($aotConfig['namespace'] ?? 'Laminas\Di\Generated') ?: 'Laminas\Di\Generated';`, but that's a matter of taste (mostly reducing number of AST nodes)

Reply:
Thanks for taking time to review.

> Is there an effective way to test this behavior?

There already is a (failing) Testcase which this PR is attempting to fix or am I missing something :thinking: ?

> Also, code above can probably be rewritten as (...)

Sweet :+1:

Reply:
@Ocramius Sorry, the proposed `$namespace = ($aotConfig['namespace'] ?? 'Laminas\Di\Generated') ?: 'Laminas\Di\Generated';` seems not to work as expected:

The ternary expression (`?: 'Laminas\Di\Generated`) catches the empty array just like `empty()` before, so the test keeps failing.

Reply:
Oh, didn't know this could be an `array`, sorry...